### ISSUE TITLE
Creating Sample Pools and positions

### ISSUE THREAD

**Issue by julianmrodri** [Uniswap/hardhat-plugin-deploy-v3#3]
It would be great to complement this with the ability to setup some Pools, initialize them, and create positions (add liquidity) so they provide a ready-to-use instance of populated Pools for other protocols to develop on top.

**Comment by julianmrodri** [comment]
Yes. It is exactly what we  need in our project. We want to integrate Uniswap V3 for doing swaps on-chain from our smart contracts and we wanted to setup a local environment to play with. 

It is quite difficult with V3, lots of details you need to learn to do that. This plugin looks good to use as a reference of which contracts you need but its just that. Doesn't take you the rest of the way. Thanks!

**Comment by quan191** [comment]
@julianmrodri i'm sorry if i disturbing you but i'm trying to re-build UniswapV3 for testing function purposes. Can I follow the plugin's step ( which contract to be deployed first) ? What should I do next to testing create pools and others functions of uniswapV3 ?
